core/sql/generator/GenRelScan.cpp

```diff
@@ -787,6 +798,7 @@ short FileScan::codeGenForHive(Generator * generator)
   const Int32 executorPredTuppIndex = 3;
   const Int32 asciiTuppIndex = 4;
   ULng32 asciiRowLen; 
+  ULng32 translateRowLen; 
```

Comment:
Looks like this is not needed? Same for transVids on line 811 and transVal on lines 1518 and 1614?